packages/admin/admin-rte/src/core/translation/ToolbarButton.tsx

```diff
@@ -1,16 +1,21 @@
 import { Tooltip, useContentTranslationService } from "@comet/admin";
 import { Translate } from "@comet/admin-icons";
+import { EditorState } from "draft-js";
 import * as React from "react";
 import { FormattedMessage } from "react-intl";
 
 import ControlButton from "../Controls/ControlButton";
 import { IControlProps } from "../types";
 import { htmlToState } from "./htmlToState";
 import { stateToHtml } from "./stateToHtml";
+import { TranslationDialog } from "./TranslationDialog";
 
 function ToolbarButton({ editorState, setEditorState, options }: IControlProps): React.ReactElement {
     const translationContext = useContentTranslationService();
 
+    const [open, setOpen] = React.useState<boolean>(false);
+    const [translationEditorState, setTranslationEditorState] = React.useState<EditorState | undefined>(undefined);
```

Comment:
Naming: `pendingEditorState` and `setPendingEditorState`

Reply:
d2b55062f472ac9d5f076a912529e170af6326ef

Reply:
I wanted to rename `translationEditorState`, not `open` 😂